app/models/comment.rb

```diff
@@ -0,0 +1,79 @@
+# == Schema Information
+#
+# Table name: comments
+#
+#  id               :bigint(8)        not null, primary key
+#  commentable_id   :integer
+#  commentable_type :string
+#  title            :string
+#  body             :text
+#  subject          :string
+#  user_id          :integer          not null
+#  parent_id        :integer
+#  lft              :integer
+#  rgt              :integer
+#  created_at       :datetime         not null
+#  updated_at       :datetime         not null
+#
+
+class Comment < ActiveRecord::Base
+  # extends ...................................................................
+  # class methods .............................................................
+  # includes ..................................................................
+
+  # relationships .............................................................
+  belongs_to :commentable, polymorphic: true
+  belongs_to :user
+
+  # validations ...............................................................
+  validates :body, presence: true
+  validates :user, presence: true
+
+  # callbacks .................................................................
+
+  # scopes ....................................................................
+
+  # Helper class method to lookup all comments assigned
+  # to all commentable types for a given user.
+  scope :find_comments_by_user, lambda { |user|
+    where(user_id: user.id).order("created_at DESC")
+  }
+
+  # Helper class method to look up all comments for
+  # commentable class name and commentable id.
+  scope :find_comments_for_commentable, lambda { |commentable_str, commentable_id|
+    where(commentable_type: commentable_str.to_s, commentable_id: commentable_id).order("created_at DESC")
+  }
```

Comment:
Love these scopes. I'd probably remove the prefix of `find_comments_` as its somewhat redundant and I think the semantics will read better. 

Consider this usage:

```ruby
Comment.by_user(user)
# ~vs~
Comment.find_comments_by_user(user)
```

Also, switch to stabby lambda `->` for scopes.

Reply:
These are gem-generated files

Reply:
I'll update it